sbysrc/sby_core.py

```diff
@@ -268,37 +268,41 @@ def parse_config(self, f):
                 continue
             match = re.match(r"^\s*\[(.*)\]\s*$", line)
             if match:
-                entries = match.group(1).split()
+                entries = match.group(1).split(" ", maxsplit = 1)
```

Comment:
Changing the split separator from the default of `None` to `" "` changes behavior regarding tabs, leading and trailing whitespace and using multiple spaces in a row. I think in the places where we expect at most a single argument that may contain inner whitespace we should use `.strip().split(maxsplit=1)` instead, so we don't change how anything currently supported is parsed.

Reply:
Ah, right, I didn't think about someone putting tabs in the section header for arguments.

I don't have any gripes with stripping before the split either, so I'll quickly change that.